### DISCUSSION TITLE
Drag and drop with connectedSortable

### DISCUSSION THREAD

**Discussion by congvc96** [ColorlibHQ/AdminLTE#3899] [Q&A]
Hi, I'm a student. I having a problem with getting data from drag and drop. I have 4 columns like in the picture, every column both have class "connectedSortable', so that I can drag and drop items between those columns. But now I want to get data on the item after I drop item into a column, I had tried so many ways on the internet but nothing worked. And then I find out that class "connectedSortable" not allowed me to interfere or prevent me from running more functions. Can someone guide me to get data from the item after I drop the item in another column.

![image](https://user-images.githubusercontent.com/55235971/130107277-ba01db38-c0a2-414c-98d8-44613f836a61.png)

**Comment by REJack** [comment]
That's a question for the Plugin that you use for DragNDrop, we didn't developed it. We only use them :D

**Reply by congvc96** [reply]
I see what you mean. Thanks for the answer. But I think I will wait for some more results.

**Reply by REJack** [reply]
alright, may some one can help you with that, I personally don't use the drag and drop plugin that's added in AdminLTE, I could help you more with Sortable.JS or Dragula ^^

**Reply by congvc96** [reply]
I never used Sortable.JS or Dragula before and I don't know how to use them. Can you help me with that? Honestly, I rarely work with front-end, so I don't know which one is good.
